### ISSUE TITLE
Not Listening ver 10.0.19041.1202 KB5005101

### ISSUE THREAD

**Issue by marzio7** [stascorp/rdpwrap#1516]
Same issue reported for version 10.0.19041.1081

same problem Not listening and access denied

same in windows pro and wuindows home

**Comment by BrianDent** [comment]
See #1509 for 19041.1202 config

**Comment by marzio7** [comment]
First of all thanks for the reply.

unfortunately I had already read and tried  #1509 without success.

![immagine](https://user-images.githubusercontent.com/90353384/132743592-698386d3-86d6-4967-9361-510eaf6802b6.png)

The .ini for 19041.1202 config is correct and fully supported but I don't understand what's wrong with that, before the update KB5005101 the replacing only the correct .ini file has always worked in both windows pro and windows home.

Uninstalling the update KB5005101 and restoring the previous .ini file rdp wrapper also goes into 'Not Linstenig' mode (I tried only in Win Pro)

![immagine](https://user-images.githubusercontent.com/90353384/132744682-7ab593ab-6a2f-4611-b69a-ecfcd1257eac.png)

**Comment by vikinglee16** [comment]
you can try this in CMD
1. net stop termservice
（1.5 edit your _rdpwrapper.ini file_ #1509 ）
2. net start termservice

**Comment by enzingerm** [comment]
Did you put a newline at the end of rdpwrap.ini?

**Comment by marzio7** [comment]
thanks all
net stop and start: tried, no effect
put a newline at the end of rdpwrap.ini: tried, no effect

**Comment by profiterolhome** [comment]
same problem here

**Comment by remmahad** [comment]
https://github.com/stascorp/rdpwrap/issues/1509


Worked perfect for me

**Comment by Tankgq** [comment]
I had the same problem.
Add a newline at the end of rdpwrap.ini is worked for me, maybe you can try adding two or more newline.

**Comment by IljaM13** [comment]
> #1509
> 
> Worked perfect for me

you forget there is also x86 (like me) ... as usual!
so... any settings for x86?

**Comment by malicon** [comment]
Those of you who get an "Acces denied" or something like, may try this.

To discard (or rename) the INI file already in C:\Program Files\RDPWrap, Terminal services must be stopped cause it opens that INI file.  Since a year or so, i found out that "Net stop termservice" failed to stop the service.  Press Start button, type "services", then run the app. Go to "Remote Desktop Services"  from there you can see the status and stop or start it at will. You may find out, like I did, that the service is'nt stopped.  The Services.msc app won't do better...
* One needs to first uninstall RDPWrapper using "uninstall.bat" (with admin priv).   Then do 
* "Net stop termservice"  That puts termservice to a stop.  BUT uninstalling discards Program Files\RDPWrap folder.  Thus one must run
* "install.bat" to create a new  RDPWrap folder among others things. It also starts Terminal Services although the BAT file complains termserv.dll version is not suppported.  So one needs to stop it again to release the INI file .
* "Net stop termservice". 
* Now the INI file can be replaced by the more up to date file you've prepared.  Dont forget, it needs an empty line at the end.  Once the proper INI file is in place, enter the command
* "Net start termservice".  It's time to check if it worked
* "RDPConf.exe"   and it may work now.

**Comment by profiterolhome** [comment]
> #1509
> 
> Worked perfect for me

Me too

**Comment by marzio7** [comment]
Thanks malicom, thank all.

Nothing, same issue.

At this point, I think it's due to some my OS setting triggered by the update.

I will try a fress install but not immediately
